langserver/build_context.go

```diff
@@ -94,7 +107,13 @@ func ContainingPackage(bctx *build.Context, filename string) (*build.Package, er
 	if PathHasPrefix(filename, bctx.GOROOT) {
 		srcDir = bctx.GOROOT // if workspace is Go stdlib
 	} else {
-		srcDir = bctx.GOPATH
+		srcDir = "" // with no GOPATH, only stdlib will work
```

Comment:
Yo! I checked the current behavior if the langserver-go has no GOPATH, and it ends up trying to join the empty string to src/, so I made sure we are failing equally ungracefully.

I think alternatively, we could just return an error here if len(gopaths)==0, "You are trying to use the go langserver without a GOPATH on a non standard library" message. I don't know enough about the surrounding and subsequent calls to know if failing here would be preferred.